apps/dialtone-documentation/docs/tokens/index.md

```diff
@@ -3,52 +3,11 @@ title: Tokens
 description: Multi-platform values that make up Dialtone's design language
 ---
 
-## Introduction
-
 Design tokens are all the defined values that are part of the design system, such as color, spacing, typography, and more. Design tokens are published in many different forms so they can be consumed on all platforms (CSS, LESS, Android, iOS, etc).
 
 See the [dialtone-tokens repository](https://github.com/dialpad/dialtone-tokens) for usage info.
```

Comment:
mmm.. should be this I guess `https://github.com/dialpad/dialtone/pkgs/npm/dialtone-tokens` or.. directly to `https://github.com/dialpad/dialtone` monorepo.

Reply:
The subrepo for tokens still has it's own readme

https://github.com/dialpad/dialtone/tree/staging/packages/dialtone-tokens